### DISCUSSION TITLE
Assignment 0-1 is due on September 21 (Extended: September 25)

### DISCUSSION THREAD

**Discussion by ibnesayeed** [cs531-f22/cs531-f22.github.io#12] [Assignment]
[Assignment 0-1](https://cs531-f22.github.io/assignments/assignment-0-1/) is due on `2022-09-21`. This is an extra points assignment in which you are asked to write an echo back server. have a working code with corresponding `Dockerfile` in your private repositories before the deadline to avoid any late penalties.

Feel free to post any questions or concerns you might have regarding this assignment in this thread.

**Comment by JMNewsome** [comment]
How is this assignment ment to be turned in?

**Reply by JMNewsome** [reply]
Also, I tried to access http://cs531.cs.odu.edu/ for the tester and the link appears to not be working. I have completed my echo server file and pushed it to my private repo. I assume that is how we mark our assignment complete? If not, please let me know the proper way of turning in this assignment as I did not see any other relevant information in the notes. 

Thank you!

**Comment by ibnesayeed** [comment]
> How is this assignment ment to be turned in? - @JMNewsome

Pushing your changes into your private repo is the way to go. However, to allow you to make changes in your code base towards the next assignment, I have added the following line in the [Assignment 0-1](https://cs531-f22.github.io/assignments/assignment-0-1/) description:

> * [Release](https://help.github.com/en/articles/creating-releases) your implementation with the `echo` tag

**Comment by ibnesayeed** [comment]
> Also, I tried to access http://cs531.cs.odu.edu/ for the tester and the link appears to not be working.

I am still in the process of getting a DNS record from the CS Systems Group for this link to work. Meanwhile, you can access it from [a temporary alternate link](http://cs531.ibnesayeed.com/). This interface is a convenient option, but not necessary for testing and submission of assignments. I have demonstrated how to use the test suite locally yesterday during the lecture.

**Comment by ibnesayeed** [comment]
All, I have extended the deadline of the echo back server assignment to this weekend (as discussed during the lecture).

However, I would encourage everyone to wrap this assignment up quickly and move to the Assignment 1, which is due next week.

**Comment by JMNewsome** [comment]
Thank you for the information

I have resubmitted my echo server under the tag "echo" and have tested it with the link provided. Thank you!
